forc/src/ops/forc_build.rs

```diff
@@ -57,16 +57,24 @@ pub fn build(command: BuildCommand) -> Result<pkg::Compiled> {
 
     // If necessary, construct a new build plan.
     let plan: pkg::BuildPlan = plan_result.or_else(|e| -> Result<pkg::BuildPlan> {
-        println!("  Creating a new `Forc.lock` file");
-        println!("    Cause: {}", e);
+        if !command.silent_mode {
```

Comment:
The issue mentioned a `verbose` flag. I didn't see that but saw this `silent_mode` flag which I'm assuming is orthogonal to `verbose`

Reply:
I think we can omit the hiding behaviour altogether in favour of moving the "cause" onto the same line as you have done below.

Reply:
Also w.r.t. `silent_mode` flag - I believe this is more about silencing output altogether (I think a `quiet` flag is a more common name in unix tooling). That said, I'm not sure we actually abide by this flag much at all yet. We should revisit the flag when we have a nicer approach to logging #884.